config/tests/snapshots/config_tests__worker_cache.snap

```diff
@@ -6,87 +6,103 @@ expression: worker_cache
   "workers": {
     "ildi4hrBzbOHBELHe0w69Yx87bh3nQJw5tTx4vc2fXQ=": {
       "0": {
+        "name": "zGIzLjS7LVzWn2Dvuyo2y5FsfrRYMB6jZjbE27ASvYg=",
         "transactions": "/ip4/127.0.0.1/tcp/2/http",
         "worker_to_worker": "/ip4/127.0.0.1/tcp/3/http",
         "primary_to_worker": "/ip4/127.0.0.1/tcp/1/http"
       },
       "1": {
+        "name": "ildi4hrBzbOHBELHe0w69Yx87bh3nQJw5tTx4vc2fXQ=",
```

Comment:
This doesn't quite look right but this is in a snap-shot file so maybe its just that?

in particular the pubkey for worker 1 here is the same as the primary and you can see that the pubkeys repeat for other workers for other primaries. They'll all need to be unique from each other.

Reply:
Thanks for fixing it for me 😄